sv_pipeline/genome/load_data.py

```diff
@@ -0,0 +1,205 @@
+            missing_sample_message = 'Missing the following {} samples:\n{}'.format(
+                len(missed_samples), ', '.join(sorted(missed_samples))
+            )
+            if ignore_missing_samples:
+                logger.info(missing_sample_message)
+            else:
+                logger.error(missing_sample_message)
+                raise Exception(missing_sample_message)
+
+        sample_remap = get_sample_remap(project_guid, sample_type)
+        message = 'Subsetting to {} samples'.format(len(sample_subset))
+        if sample_remap:
+            message += ' (remapping {} samples)'.format(len(sample_remap))
+            sample_subset = sample_subset - set(sample_remap.keys())
+            sample_subset.update(set(sample_remap.values()))
+            mt = mt.key_cols_by()
+            sample_remap = hl.literal(sample_remap)
+            mt = mt.annotate_cols(s=hl.if_else(sample_remap.contains(mt.s), sample_remap[mt.s], mt.s))
+        logger.info(message)
+
+        mt = mt.filter_cols(hl.literal(sample_subset).contains(mt.s))
+
+    genotypes = hl.agg.collect(hl.struct(sample_id=mt.s, gq=mt.GQ, num_alt=mt.GT.n_alt_alleles(), cn=mt.RD_CN))
+    mt = mt.annotate_rows(genotypes=genotypes)
+    return mt.filter_rows(mt.genotypes.any(lambda x: x.num_alt > 0)).rows()
+
+
+def annotate_fields(rows, gene_id_mapping):
+    rows = rows.annotate(**{k: v(rows) for k, v in BASIC_FIELDS.items()})
+
+    rows = rows.annotate(
+        gene_affected=hl.filter(
+            lambda x: hl.is_defined(x),
+            [rows.info[col].map(lambda gene: hl.struct(gene_symbol=gene, gene_id=gene_id_mapping[gene],
+                                                       predicted_consequence=col.split('__')[-1]))
+             for col in [gene_col for gene_col in rows.info if gene_col.startswith('PROTEIN_CODING__')
+                         and rows.info[gene_col].dtype == hl.dtype('array<str>')]]),
+        sv_type=rows.alleles[1].replace('[<>]', '').split(':', 2),
+    )
+
+    COMPUTED_FIELDS.update({'filters': lambda rows: hl.if_else(hl.len(rows.filters) > 0, rows.filters,
+                                                               hl.null(hl.dtype('array<str>')))})
+    rows = rows.annotate(**{k: v(rows) for k, v in COMPUTED_FIELDS.items()})
+
+    rows = rows.rename({'rsid': 'variantId'})
+
+    return rows.key_by('variantId').select(*FIELDS)
+
+
+def export_to_es(rows, input_dataset, project_guid, es_host, es_port, block_size, num_shards):
+    meta = {
+      'genomeVersion': '38',
+      'sampleType': WGS_SAMPLE_TYPE,
+      'datasetType': 'SV',
+      'sourceFilePath': input_dataset,
+    }
+    index_name = get_es_index_name(project_guid, meta)
+
+    es_password = os.environ.get('PIPELINE_ES_PASSWORD', '')
+    es_client = ElasticsearchClient(host=es_host, port=es_port, es_password=es_password)
+
+    es_client.export_table_to_elasticsearch(
+        rows,
+        index_name=index_name,
+        index_type_name='_doc',
```

Comment:
don't override default parameters, it makes the code harder to read

Reply:
Deleted this line.